### DISCUSSION TITLE
Node.JS Usage?

### DISCUSSION THREAD

**Discussion by IzMichael** [supabase/supabase#2778] [Questions] [answered]
Hiya, 

I'm trying to figure out how to use the realtime API from a Node.js project (Discord.JS Bot).
I've tried using the NPM module, but I just get the following error:
![image](https://user-images.githubusercontent.com/57115702/128690709-71b983c7-477e-4541-b8f9-ea4eacfecddd.png)

Any help will be appreciated :D

**Comment by silentworks** [comment, marked as answer]
From your error, it seems you are trying to do an import in a commonjs project. You probably need to use:

```js
const { createClient } = require('@supabase/supabase-js');
```
